app/src/main/java/org/p2p/wallet/history/repository/HistoryTransactionConverter.kt

```diff
@@ -18,10 +21,10 @@ import org.threeten.bp.ZoneId
 import org.threeten.bp.ZonedDateTime
 import java.math.BigDecimal
 
-object TransactionConverter {
+class HistoryTransactionConverter {
 
     /* Swap transaction */
-    fun fromNetwork(
+    fun mapSwapTransactionToHistory(
```

Comment:
Did we decide to move from `fromNetwork`/`toNetwork` style?

Reply:
it's not about network here, but about mapping domain entity to another domain entity